estate/models/estate_property.py

```diff
@@ -0,0 +1,56 @@
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'The Real Estate Advertisement module'
+
+    name = fields.Char('Name', required=True)
+    description = fields.Text('Description')
+    postcode = fields.Char('Postcode')
+    bedrooms = fields.Integer('Bedrooms', default=2)
+    living_area = fields.Integer('Living Area (aqrm)')
+    facades = fields.Integer('Fcadres')
+    garage = fields.Boolean(default=False)
+    garden = fields.Boolean(default=False)
+    garden_area = fields.Integer('area', default=0)
+    expected_price = fields.Float('Expected Price', required=True)
+    active = fields.Boolean(default=True)
+    sales_man = fields.Many2one("res.users", string="Salesmam", default=lambda self: self._uid)
+    buyer = fields.Many2one("res.partner", string="Buyer", copy=False)
+    type = fields.Many2one("estate.property.type", string="Type")
+    tag = fields.Many2many("estate.property.tag", string="Tags")
+    offer_ids = fields.One2many("estate.property.offer", "")
+    date_availability = fields.Date(
+        'Availability Date',
+        copy=False,
+        default=fields.Datetime.add(fields.Datetime.today(), months=3)
+    )
+
+    selling_price = fields.Float(
+        'Selling Price',
+        readonly=True,
+        copy=False
+        )
+
+    garden_orientation = fields.Selection(
+        [
+            ('North', 'North'),
+            ('South', 'South'),
+            ('East', 'East'),
+            ('West', 'West')
+        ],
+        string='Garden Orientation'
+    )
+    state = fields.Selection(
+        [
+            ("New", "New"),
+            ("Offer Received", "Offer Received"),
+            ("Offer Accepted", "Offer Accepted"),
+            ("Sold", "Sold"),
+            ("Canceled", "Canceled")
```

Comment:
```suggestion
            ("new", "New"),
            ("offer_received", "Offer Received"),
            ("offer_accepted", "Offer Accepted"),
            ("sold", "Sold"),
            ("canceled", "Canceled")
```
The _keys_ are better lower cased with underscore between them to really see the diff between the key and the value